src/richie-front/js/components/SearchFilter/SearchFilter.tsx

```diff
@@ -15,12 +15,13 @@ export const SearchFilter = (props: SearchFilterProps) => {
 
   return (
     <button
-      className={`search-filter ${isActive ? 'active' : ''}`}
+      className={`search-filter ${isActive ? 'active close' : ''}`}
```

Comment:
What's the purpose of this `close` class?

Reply:
It comes from the bootstrap documentation. It applies style on the button https://getbootstrap.com/docs/4.1/utilities/close-icon/

I see you use bootstrap on this project.

Reply:
Ok. We're not using bootstrap directly, just `@import`ing some stuff we need. Also those are not buttons in the style of bootstrap as you can probably see.

Additionally following an impetus from @sveetch we have been dropping bootstrap from our styles.

Reply:
We're mostly following BEM. There's no proper style guide outside of that. I think we're in the middle of a process to move from bootstrap to our own thing.